sdk/python/resources/compute/attach_manage_spark_pools.ipynb

```diff
@@ -107,13 +111,14 @@
     "    DefaultAzureCredential(), subscription_id, resource_group, workspace_name\n",
     ")\n",
     "\n",
-    "synapse_name = \"<ATTACHED_SPARK_POOL_NAME>\"\n",
+    "synapse_name = \"<ATTACHED_SPARK_POOL_NAME_UAI>\"\n",
     "synapse_resource = \"/subscriptions/<SUBSCRIPTION_ID>/resourceGroups/<RESOURCE_GROUP>/providers/Microsoft.Synapse/workspaces/<SYNAPSE_WORKSPACE_NAME>/bigDataPools/<SPARK_POOL_NAME>\"\n",
     "synapse_identity = IdentityConfiguration(\n",
     "    type=\"UserAssigned\",\n",
     "    user_assigned_identities=[\n",
-    "        UserAssignedIdentity(\n",
-    "            resource_id=\"/subscriptions/<SUBSCRIPTION_ID/resourceGroups/<RESOURCE_GROUP>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/<AML_USER_MANAGED_ID>\"\n",
+    "        ManagedIdentityConfiguration(\n",
+    "            client_id=\"<USER_ASSIGNED_IDENTITY_CLIENT_ID>\",\n",
+    "            resource_id=\"/subscriptions/<SUBSCRIPTION_ID>/resourceGroups/<RESOURCE_GROUP>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/<AML_USER_MANAGED_ID>\",\n",
```

Comment:
Can we define a separate variable:
```python
# Define name of user-assigned managed identity
user_managed_id = "<AML_USER_MANAGED_ID>"
# Define Client ID for user-assigned managed identity
user_managed_client_id = "<USER_ASSIGNED_IDENTITY_CLIENT_ID>"
```
And then change these two line of code to:
```python
client_id=f"{user_managed_client_id}",
resource_id=f"/subscriptions/{subscription_id}/resourceGroups/{resource_group}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{user_managed_id}",
```

Reply:
done